### PR TITLE
fix(suite-native): feature flag in crossroads condition

### LINKED ISSUE #10088: iOS app displays content of Android
Steps to reproduce
If you unistall and install iOS app on DEV (app version: 23.11.1 (382)), you see content which should be on Android (connected Trezor features)

See the printscreen:
Image

### PR DESCRIPTION
**Pull request by juriczech**
## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/10088

### CODE REVIEW

**Review by PeKne — APPROVED** [review]
just a nit

File: suite-native/module-home/src/components/EmptyHomeRenderer.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const EmptyHomeRenderer = () => {
     );
 
     // Crossroads should be displayed only if there is no real device connected and portfolio tracker has no accounts.
-    if (areAllDevicesDisconnectedOrAccountless) {
+    if (areAllDevicesDisconnectedOrAccountless && isUsbDeviceConnectFeatureEnabled) {
```
Comment (PeKne) [inline suite-native/module-home/src/components/EmptyHomeRenderer.tsx]: the `isUsbDeviceConnectFeatureEnabled` is checked also in the following if. What about to wrap these two conditions to 
```tsx
if (isUsbDeviceConnectFeatureEnabled){}
```

Comment (juriczech) [reply suite-native/module-home/src/components/EmptyHomeRenderer.tsx]: https://github.com/trezor/trezor-suite/pull/10090/commits/2e5d97e160cf9d3bb28157481f3c5e13b3f18ecf

**Pull request merged**